README.md

```diff
@@ -238,6 +238,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 ### Security
 
 * [Aqua Security 💲](https://www.aquasec.com) - Securing container-based applications from Dev to Production on any platform
+* [bane](https://github.com/jessfraz/bane) - Custom & better AppArmor profile generator for Docker container by [@jessfraz][jessfraz]
```

Comment:
What does `better` mean ? Please be specific. Better than what and how ?
Also, a description in the PR would be nice 😄